### DISCUSSION TITLE
[Bug]: @pageNumber and @totalPages not showing

### DISCUSSION THREAD

**Discussion by aldinokemal** [spatie/laravel-pdf#100] [Q&A]
### What happened?

I tried using this in my footer view (->footerView('public::public.pdf.footer'))

```
<div style="text-align: center; font-size: 10px; color: #000; padding-left: 20px">
    @pageNumber dari @totalPages
</div>
```

<img width="207" alt="Screenshot 2024-03-08 at 23 23 22" src="https://github.com/spatie/laravel-pdf/assets/14232125/3135b77a-615f-41ad-8ad1-501ebd188702">

### How to reproduce the bug

I'm using browser `https://storage.googleapis.com/chrome-for-testing-public/122.0.6261.111/linux64/chrome-headless-shell-linux64.zip` that running in docker ubuntu

### Package Version

1.4.0

### PHP Version

8.2.0

### Laravel Version

10.0

### Which operating systems does with happen with?

macOS, Linux

### Notes

Running inside docker
- Node v20

**Comment by dgironella** [comment]
I have same problem, I change to use another version of chrome solve it. Now I use chrome/linux-122.0.6261.111/chrome-linux64/chrome and pageNumber and totalPage work.

**Comment by TomOusselin** [comment]
Same problem here...

![image](https://github.com/spatie/laravel-pdf/assets/76063513/e6ffd308-1c42-447f-9ac3-2208ac0c1b78)

<div style="text-align: center; font-size: 10px; color: #000; padding-left: 20px">
    @pageNumber of @totalPages
</div>

How can i change my chrome version ?

**Comment by kaspernowak** [comment]
@aldinokemal Instead of using manually downloaded chrome, you can use this approach instead: https://github.com/spatie/laravel-pdf/discussions/134#discussioncomment-9309129

@TomOusselin You can either update your version of puppeteer or download chrome manually.
If you download it manually, you need to set the path to the new version like this: https://spatie.be/docs/browsershot/v4/requirements#content-custom-chromechromium-executable-path

**Comment by bagusmahendra85** [comment]
my footer is incredibly small, are you guys too ?
